src/wallet/rpcwallet.cpp

```diff
@@ -243,16 +254,24 @@ UniValue getrawchangeaddress(const JSONRPCRequest& request)
         pwallet->TopUpKeyPool();
     }
 
+    OutputType output_type = g_change_type;
```

Comment:
Mirror argument name:
```cpp
OutputType address_type = g_change_type;
```

Reply:
Rewrote this code.

Reply:
I'm using 'output type' as the internal name, 'address type' is just used for the global/cmdline.

Reply:
> Part of the reason for that is that I feel using the term 'address' for change is wrong

For that reason rename argument above from `address_type` to `change_type`?